src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -9,180 +9,387 @@
+        // Shortests symbol title have preference over larger titles
+        if (a.inputLengthDifference > b.inputLengthDifference) return 1;
+        if (a.inputLengthDifference < b.inputLengthDifference) return -1;
         return 0;
       });
     },
-  },
-  watch: {
-    userInput: 'debounceInput',
+    scrollIntoView() {
+      this.$refs.match[this.focusedIndex].scrollIntoView({
+        block: 'nearest',
+        inline: 'start',
+      });
+    },
+    startingPointHook() {
+      this.focusedIndex = this.totalItemsToNavigate - 1;
+    },
   },
 };
 </script>
 
-<style scoped lang="scss">
+<style lang="scss">
 @import 'docc-render/styles/_core.scss';
 
-  .close-icon {
-    padding-bottom: 10px;
+$modal-margin-top: 10rem;
+$base-border-width: 1px;
+$filter-padding: rem(15px);
+
+.quick-navigation {
+  input[type="text"] {
+    @include font-styles(body-large);
   }
-  .container {
-      flex-direction: row;
+  &__container {
+    background-color: var(--color-fill);
+    border: solid $base-border-width var(--color-fill-gray);
+    border-radius: $border-radius;
+    filter: drop-shadow(0px 7px 50px rgba(0, 0, 0, 0.25));
+    margin: auto;
+    max-width: rem(800px);
+    .filter__wrapper {
+      border: none;
+    }
+    > * {
+      --input-text: var(--color-figure-gray-secondary) !important;
+    }
   }
-  .filter {
+  &__filter {
+    background: var(--color-fill);
+    border: none;
     border-radius: $border-radius;
-    background-color: var(--color-fill);
-    border: solid var(--color-figure-gray) 1px;
-    padding: 20px;
-    width: 80%;
-    margin-bottom: 20px;
+    box-sizing: border-box;
+    outline-width: 0;
+    width: 100%;
   }
-  .quick-navigation-modal {
-    background-color: var(--color-fill);
+  &__magnifier-icon-container {
+    display: flex;
+    height: auto;
+    width: rem(18px);
+    margin: auto;
+    padding-left: rem(5px);
+    > * {
+      width: 100%;
+    }
+    &.blue .magnifier-icon {
+      fill: var(--color-figure-blue);
+      color: var(--color-figure-blue);
+    }
+  }
+  &__match-list {
+    overflow: scroll;
+    max-height: rem(450px);
+    &.active {
+      border-top: 1px solid var(--color-fill-gray);
+    }
+    .no-results {
+      margin: $filter-padding auto $filter-padding auto;
+      width: fit-content;
+    }
+    .selected {
+      background-color: var(--color-navigator-item-hover);
+    }
+  }
+  &__modal-shadow {
+    background-color: var(--color-quick-navigation-modal-shadow);
     position: fixed;
-    top: 0;
     bottom: 0;
     left: 0;
     right: 0;
-    z-index: 100;
-    padding: 100px;
-    overflow: scroll;
+    top: 0;
+    padding: min(rem(10px));
+    padding-top: $modal-margin-top;
+  }
+  &__open-key-container {
+    margin-top: auto;
+    margin-bottom: auto;
+    margin-right: rem(15px);
+    color: var(--color-figure-gray-secondary);
   }
-  .symbol-match {
-    width: fit-content;
+  &__reference:hover {
+      text-decoration: none;
+    }
```

Comment:
```suggestion
  }
```

Reply:
Resolved in https://github.com/apple/swift-docc-render/pull/396/commits/62db4a637515e8fd67f83266a028ced48f825d01